### PR TITLE
ADS: Use Destructive Dialogs

### PR DESCRIPTION
**Pull request by malmstein**
Task/Issue URL: https://app.asana.com/0/488551667048375/1204226367321025

### Description
This PR adds two Destructive Buttons to Bookmarks and AppTP

### Steps to test this PR

_Remove folder with content_
- [x] Create a folder and add a bookmark to it
- [x] In Bookmarks screen, use the overflow button to delete the folder
- [x] Verify that Destructive button is shown

_Remove AppTP feature_
- [x] Enable AppTP
- [x] In Trackers screen, verify that Disable and Delete Data text is not red
- [x] Tap on it
- [x] Verify that Destructive button is shown

### UI changes
| Before  | After |
| ------ | ----- |
Screenshot_20230404_103556|Screenshot_20230404_103839|

| Before  | After |
| ------ | ----- |
Screenshot_20230404_102934|Screenshot_20230404_103452|

### CODE REVIEW

**Comment by malmstein** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #3026** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3026" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/3026?utm_source=stack-comment).

File: app-tracking-protection/vpn-impl/src/main/res/values/strings-vpn.xml [hunk 1]
```diff
@@ -364,7 +364,7 @@
     <string name="atp_ManageRecentAppsProtectionEmpty">Apps with recently blocked tracking attempts will appear here</string>
 
     <!--  App Tracking Protection Remove Feature Dialog -->
-    <string name="atp_RemoveFeatureDialogTitle">Disable and Delete Data?</string>
+    <string name="atp_RemoveFeatureDialogTitle">Disable and delete data?</string>
```
Comment (nalcalag) [inline app-tracking-protection/vpn-impl/src/main/res/values/strings-vpn.xml:367]: This change needs to be in a different PR to trigger translations for other languages

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -603,6 +603,7 @@
     <string name="folderLocationTitle">Select Location</string>
     <string name="addFolder">Add Folder</string>
     <string name="editFolder">Edit Folder</string>
+    <string name="bookmarkFolderDeleteDialogTitle" instruction="Dialog title for deleting a folder with bookmarks">"Delete folder and its contents?"</string>
```
Comment (nalcalag) [inline app/src/main/res/values/strings.xml:606]: You can put this new string resource in donottranslate.xml and then, in another PR move it to strings.xml to trigger translations
